package.json

```diff
@@ -3,6 +3,7 @@
   "version": "0.5.18",
   "description": "Same as JSON but with added support for `Date`, `undefined`, `Map`, `Set`, and more.",
   "main": "./index.mjs",
+  "type": "module",
```

Comment:
I think this means we drop support for older Node.js versions? If they are all EOL I'm ok with it. But then we might want to remove CJS altogether, I think.

Reply:
I think that it should not break older node version, as the `main` field already pointed to a `.mjs` file. Either way, if we're not sure, we should perhaps publish it as `0.6.0` then.

Reply:
I meant `require(esm)`, which means we can now drop CJS builds since March 2025. I'll still add a `MINOR BREAKING CHANGE`.